telco-hub/configuration/reference-crs-kube-compare/required/acm/observabilityMCO.yaml

```diff
@@ -17,16 +17,22 @@ spec:
   # for an scenario with:
   # 3500SNOs, 125 pods and 4 Namespaces (apart from Openshift NS)
   # storage retention 15 days
-  # downsampling disabled
+  # downsampling enabled
   # default MCO Addon configuration samples_per_hour, pv_retention_hrs.
   # More on how to stimate: https://access.redhat.com/articles/7103886
+  # changes on the size affects to the number of replicas, that
+  # affects to cluster capacity and storage
+  # changes on the retention affects to the cluster storage
   advanced:
     retentionConfig:
       blockDuration: 2h
       deleteDelay: 48h
-      retentionInLocal: 24h
-      retentionResolutionRaw: 15d
-  enableDownsampling: false
+      retentionInLocal: {{ .spec.advanced.retentionConfig.retentionInLocal }}
+      retentionResolutionRaw: {{ .spec.advanced.retentionConfig.retentionResolutionRaw | default "15d"}}
+      retentionResolution5m: {{ .spec.advanced.retentionConfig.retentionResolutionRaw | default "15d"}}
+      retentionResolution1h: {{ .spec.advanced.retentionConfig.retentionResolutionRaw | default "15d"}}
```

Comment:
based on @lack suggestion (thanks Ian). 
I have added that, not only retention have to have the same value, but it has to be configured. Not mandatory to any value, but if does not exists, we go to the RDS default.
what do you think @imiller0 ? I dont remember if we tried the idea of regex for some more complex scenario, that now I dont have in mind.
@lack do you like this added default value?